Assets/HoloToolkit/Utilities/Scripts/Editor/SceneSettingsWindow.cs

```diff
@@ -1,23 +1,27 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using UnityEditor;
 using UnityEngine;
 
 namespace HoloToolkit.Unity
 {
     /// <summary>
-    /// Renders the UI and handles update logic for HoloToolkit/Configure/Apply Mixed Reality Scene Settings.
+    /// Renders the UI and handles update logic for MixedRealityToolkit/Configure/Apply Mixed Reality Scene Settings.
     /// </summary>
     public class SceneSettingsWindow : AutoConfigureWindow<SceneSettingsWindow.SceneSetting>
     {
+        /// <summary>
+        /// Can be found in the meta file of the camera prefab.  We use the GUID instead of a path in case people move the toolkit folders & assets around in their own projects.
+        /// </summary>
+        private const string CameraPrefabGUID = "f46f9eddea7bd4d4e813283cdc60cb96";
```

Comment:
This is the HoloLensCamera prefab GUID. What about the other camera prefabs?

We should probably go down to 1 prefab anyway, which adapts to any situation.

Reply:
Pretty sure I updated the GUID to point to be the MixedRealityCamera prefab.

Reply:
@keveleigh would you prefer the `MiedRealityCamera` prefab, or the `MixedRealityCameraParent` prefab?

Reply:
Weird, it was definitely a different GUID when I commented, though my window must have been out of date!

If we want to go down to 1 prefab, reusing the HoloLensCamera GUID would probably be best to maintain forward compatibility, right?

Reply:
>If we want to go down to 1 prefab, reusing the HoloLensCamera GUID would probably be best to maintain forward compatibility, right?

I don't think so.  I'd fear the new branch would get a lot of missing references if we changed that.

I think it's fine the way it is anyway, cause the scene settings window will just replace the camera in the scene anyway.  That way devs have the option to update, or use the new prefab.

Reply:
Cool, in that case, MixedRealityCameraParent would be the preferred one to use.

Reply:
I'll update the GUID to point to it.